packages/preview/guided-resume-starter-cgc/2.0.0/template/templates/resume.template.typ

```diff
@@ -0,0 +1,148 @@
+#let resume(
+  author: "",
+  location: "",
+  contacts: (),
+  body
+) = {
+  set document(author: author, title: author)
+  set text(
+    font: "New Computer Modern",
+    size: 11pt,
+    lang: "en"
+  )
+  set page(
+    margin: (
+      top: 1.25cm,
+      bottom: 0cm,
+      left: 1.5cm,
+      right: 1.5cm
+    ),
+  )
+
+  show link: set text(
+    fill: rgb("#0645AD")
+  )
+  
+  show heading: it => [
+    #pad(top: 0pt, bottom: -15pt, [#smallcaps(it.body)])
+    #line(length: 100%, stroke: 1pt)
+  ]
+
+  // Author
+  align(center)[
+    #block(text(weight: 700, 2.5em, [#smallcaps(author)]))
+  ]
+
+  // Contact information.
+  pad(
+    top: 0.25em,
+    align(center)[
+      #smallcaps[#contacts.join("  |  ")]
+    ],
+  )
+
+  // Location
+  if location != "" {
+    align(center)[
+      #smallcaps[#location]
+    ]
+  }
+
+  // Main body.
+  set par(justify: true)
+
+  body
+}
+
+// Allows hiding or showing full resume dynamically using global variable
+#let hide(shouldHide, content) = {
```

Comment:
Typst packages generally use kebab-case for naming functions and their arguments. But this is not a hard requirement, and you can decide to leave this as is if you prefer.
```suggestion
#let hide(should-hide, content) = {
```

Reply:
This should be fixed in [`49d62cd`](https://github.com/typst/packages/pull/580/commits/f9d62cdadd5ab001bb625970125161d153255840)!